controls/roles/validator-import/molecule/default/molecule.yml

```diff
@@ -0,0 +1,30 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: hetznercloud
+platforms:
+  - name: "validator--import--default--ubuntu-20.04"
```

Comment:
`validator-import--default--ubuntu-20.04` the separator is always two minus like this: `--`

Reply:
okey, got it